velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -425,21 +426,34 @@ void DuckDbQueryRunner::createTable(
   }
 }
 
+void DuckDbQueryRunner::initializeTpch(double scaleFactor) {
+  db_.LoadExtension<::duckdb::TPCHExtension>();
+  auto query = fmt::format("CALL dbgen(sf={})", scaleFactor);
+  execute(query);
+}
+
+void verifyDuckDBResult(DuckDBQueryResult& result, std::string_view sql) {
```

Comment:
- result can be passed as a const reference, e.g. const DuckDBQueryResult& result
- put this function into an anonymous namespace